package/MDAnalysis/analysis/rms.py

```diff
@@ -138,70 +137,90 @@
 logger = logging.getLogger('MDAnalysis.analysis.rmsd')
 
 
-def rmsd(a, b, weights=None, center=False):
+def rmsd(a, b, weights=None, center=True, superposition=False):
```

Comment:
I'm not sure what the done thing is with changing default keywords.  Do we have to issue some sort of warning about this?  Someone blindly using this would see their results change with no apparent reason why.